### PR TITLE
Dev autofill event creation

### PR DESCRIPTION
**Pull request by evanmielak**

### CODE REVIEW

File: views/admin/new_event.erb [hunk 1]
```diff
@@ -267,4 +268,31 @@
         }
     }).change();
 
+    $('#start-time-hour').change(function (change) {
+        start_time_hour = parseInt($('#start-time-hour').val());
+        duration = parseInt($('#duration').val());
+        time = start_time_hour + ((duration - (duration % 60)) / 60);
+        time = time % 12;
+        $('#end-time-hour').val(time);
+        console.log(time);
```
Comment (adobrusky) [inline views/admin/new_event.erb]: Remove console.log

File: views/admin/new_event.erb [hunk 2]
```diff
@@ -267,4 +268,31 @@
...
+        if (time >= 60){
+            $('#end-time-hour').val((parseInt($('#end-time-hour').val()) + 1) % 12);
+            time = time % 60;
+            $('#end-time-minute').val(time);
+        }
+        $('#end-time-minute').val(time);
+    }).change();
+
+    $('#start-date').change(function (change) {
+        console.log($('#start-date').val());
```
Comment (adobrusky) [inline views/admin/new_event.erb]: Remove console.log

**Review by adobrusky — CHANGES_REQUESTED** [review]
Besides the debugging statements that need to be removed, I'm impressed with the code, and I really like the UI for selecting the preset. 

I did notice a bug with the duration that will need to be fixed. After entering a start hour and start minute, every time you change the dropdown value for the start minute the end hour will increment. If you do this enough, the end hour dropdown is set to blank because it gets a value greater than 12 which is not supported in the dropdown. 

There is also another bug where if you go to edit an event the page crashes because it tries to fill in the hidden field that stores `duration` but since it is not being passed `duration` it is null and crashes.

<del>Also with the current duration implementation, it locks the admins into using the preset duration since the end time automatically changes when the start time changes. I think it's likely they wouldn't want to set a duration different than the preset duration (so this is probably fine), but in case they wanted to make the event shorter/longer, they would be unable to do so. It would require a bit of tweaking, but you could simplify the duration logic and give the admins the option to stray from the preset duration by adding a button under the start time inputs that, when clicked, would autofill the end time based on the preset duration. This way the admins would just enter a start time and then click a button to autofill the end time. Then you wouldn't have to handle the automatic updating (which would remove the bug I mentioned) and if the admin wanted to have a duration other than the preset duration they would be able to change the end time after autofilling. </del>

**Comment by evanmielak** [comment]
> Also with the current duration implementation, it locks the admins into using the preset duration since the end time automatically changes when the start time changes. I think it's likely they wouldn't want to set a duration different than the preset duration (so this is probably fine), but in case they wanted to make the event shorter/longer, they would be unable to do so.

If you choose the start time first, you can change the end time to anything that you want. Do we want the expected behavior to be something different?

**Review by adobrusky — APPROVED** [review]
Looks good

**Pull request merged**